src/reducers/filter.js

```diff
@@ -0,0 +1,12 @@
+import { CHANGE_FILTER } from '../actions/index';
+
+const bookFilterReducer = (state = '', action) => {
```

Comment:
- [ ] The state here should default to "All" ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/ItDefaultsToAll.png)).